ui/components/app/token-detected/token-address/token-address.scss

```diff
@@ -0,0 +1,5 @@
+.token-address {
+  & &__copy-link {
+    padding: 10px 0;
```

Comment:
Can this adhere to the 8px grid? OR does it need this 10px padding to align with other elements?
```suggestion
    padding: 8px 0;
```

Reply:
@georgewrmarshall this is how the UI changes with 8px

<img width="259" alt="Screen Shot 2022-03-22 at 11 37 37 AM" src="https://user-images.githubusercontent.com/43930900/159520624-af2f3b21-b315-4046-9b17-5558dd3b8ba8.png">

Reply:
Ah ok. I'd possibly bump up the copy font-size to 24px and use a margin-left of 16px